test/agrovoc-allegrograph_on_hold.py

```diff
@@ -215,17 +214,14 @@ def testSelectByGETinXML(self):
         result = self.__generic(selectQuery, XML, GET)
         ct = result.info()["content-type"]
         assert True in [one in ct for one in _SPARQL_XML], ct
-        results = result.convert()
-        results.toxml()
-        self.assertEqual(results.__class__.__module__, "xml.dom.minidom")
-        self.assertEqual(results.__class__.__name__, "Document")
+        results = result.convert().toxml()
+        self.assertEqual(type(results), str)
```

Comment:
Again if you could clarify the type here, why str.

Reply:
8347df7659fa64bd98bf892426e6223942dc0856